bundle/manifests/opendatahub-operator.clusterserviceversion.yaml

```diff
@@ -177,16 +409,24 @@ spec:
   - supported: true
     type: AllNamespaces
   keywords:
-  - '### Components'
+  - Operator
+  - OpenShift
+  - Open Data Hub
+  - opendatahub
   links:
-  - name: Opendatahub Operator
-    url: https://opendatahub-operator.domain
+  - name: Open Data Hub
+    url: https://opendatahub.io
+  - name: Open Data Hub Community
+    url: https://github.com/opendatahub-io
+  maintainers:
+  - email: maintainers@lists.opendatahub.io
+    name: Open Data Hub Maintainers
   maturity: stable
+  minKubeVersion: 1.22.0
   provider:
-    name: Open Data Hub operator allows users to install and manage components of
-      the Open Data Hub. Users can mix and match tools from each project to fulfill
-      the needs of their use case. Each of the
-    url: projects share some components, but can be mostly seen as an extension of
-      each other to provide a complete solution for both novice and skilled enterprise
-      users.
+    name: Open Data Hub
+  replaces: opendatahub-operator.v1.4.2
+  selector:
+    matchLabels:
+      component: opendatahub-operator
   version: 0.0.1
```

Comment:
Bundle generation takes values from environment variables. So following env variables should be set to reflect the correct values
```
export VERSION=1.5.0
export IMG=quay.io/opendatahub/opendatahub-operator:v1.5.0
```

Reply:
Good catch. Updated